.github/workflows/test-release.yml

```diff
@@ -0,0 +1,134 @@
+name: test-release
+
+on:
+  release:  # TODO this needs to be deleted
+    types: [published]
+    inputs:
+      release-pypi:
+        required: true
+        type: boolean
+        default: true
+  workflow_dispatch:
+    inputs:
+      release-pypi:
+        required: true
+        type: boolean
+        description: "if true a release is made on PyPI"
+
+jobs:
+  test:
+    uses: ./.github/workflows/test.yml
+    secrets: inherit
+
+  build:
+    needs: test
+    uses: ./.github/workflows/build.yml
+
+  docs:
+    needs: test
+    uses: ./.github/workflows/docs.yml
+    secrets: inherit
+
+  release-core-pypi:
+    needs: [ build ]
+    runs-on: ubuntu-latest
+    if: |
+      github.event.inputs.release-pypi == 'true' && ( startsWith(github.ref_name, 'fairchem_core-') || startsWith(github.event.release.tag_name, 'fairchem_core-') )
+
+    environment:
+      name: test-pypi
+      url: https://pypi.org/p/fairchem-core/
```

Comment:
Should this path be, https://test.pypi.org/p/fairchem-core/ ?

Reply:
You're right that makes more sense! I was going by just the docs for the pypi-publish action, but according to Github - that url is used to link the deployment that shows up on github